test/unittests/mongoBackend/mongoUpdateContextGeo_test.cpp

```diff
@@ -493,6 +493,313 @@ TEST(mongoUpdateContextGeoRequest, updateLocAttribute)
     utExit();
 }
 
+/* ****************************************************************************
```

Comment:
Unit test are fine but it would be great if you could also add a .test covering the same cases :) What do you think?

Reply:
Fixed in 40da980